pkg/decode/decode_protobuf.go

```diff
@@ -90,11 +90,13 @@ func PBFlowToMap(flow *pbflow.Record) config.GenericMap {
 		}
 	}
 
+	out["DnsErrno"] = flow.GetDnsErrno()
```

Comment:
As discussed, my concern here is adding unecessary field on non dns related flows.
We agreed to keep it as is for now.